src/05-derivatives/02-common-data-types.md

```diff
@@ -116,9 +116,21 @@ sub-01/func/sub-01_task-rest_space-fsLR_den-91k_bold.json
 ```JSON
 {
     "SpatialReference": {
-        "VolumeReference": "https://templateflow.s3.amazonaws.com/tpl-MNI152NLin6Asym_res-02_T1w.nii.gz",
-        "CIFTI_STRUCTURE_CORTEX_LEFT": "https://github.com/mgxd/brainplot/raw/master/brainplot/Conte69_Atlas/Conte69.L.midthickness.32k_fs_LR.surf.gii",
-        "CIFTI_STRUCTURE_CORTEX_RIGHT": "https://github.com/mgxd/brainplot/raw/master/brainplot/Conte69_Atlas/Conte69.R.midthickness.32k_fs_LR.surf.gii"
+        "VolumeReference": "bids:templateflow:/tpl-MNI152NLin6Asym_res-02_T1w.nii.gz",
```

Comment:
I am not entirely sure whether this would work. Help would be appreciated, cc @effigies

Reply:
Templateflow is not BIDS. I think a plain URL is preferable to shoehorning `bids:` schema URIs here.

Reply:
If you look a bit below you see the mapping from `bids:templateflow:` to `https://templateflow.s3.amazonaws.com`.

I feel like this would allow us to write a shorter specification of what is allowed and when: All pointers must be BIDS URIs, instead of: sometimes they can also be `https`, etc.

WDYT?

Reply:
I think BIDS URIs solve a specific problem of allowing unambiguous relative paths that may cross dataset boundaries, but I don't think we should be rewriting all possible URIs this way. It dilutes the meaning of `bids:` and adds unnecessary indirection when a normal URI suffices.

One thing I like about the BIDS URI approach is that, by making it a URI, fields that permit *any* URI can now link BIDS datasets. So we can just say that these are URIs and that relative paths are deprecated. Will make a suggestion...

Reply:
thanks - I'll keep this example as URI then